contrib_bots/lib/DefineBot/docs.md

```diff
@@ -0,0 +1,20 @@
+# DefineBot
```

Comment:
This file is not rendered properly according to the markdown syntax. Could you check it?